src/features/smartSearch/components/filters/SurveyOption/index.tsx

```diff
@@ -215,11 +215,13 @@ const SurveyOption = ({
                 onChange={(e) => handleConditionSelectChange(e.target.value)}
                 value={filter.config.operator}
               >
-                {Object.values(CONDITION_OPERATOR).map((o) => (
-                  <MenuItem key={o} value={o}>
-                    <Msg id={localMessageIds.conditionSelect[o]} />
-                  </MenuItem>
-                ))}
+                {Object.values(CONDITION_OPERATOR)
+                  .filter((o) => o !== CONDITION_OPERATOR.SOME)
```

Comment:
This is a nice and quick way of removing it from the UI, but it doesn't remove it from the type or underlying data structures.

Reply:
Yes, I see the problem. When I read the issue, I interpreted it as a question of simply preventing the regular user from interacting with the invalid operator. I can certainly take a look at it, but might need to bow out, or enlist some help at the next hackathon, if it turns out to be too much for my level.